### ISSUE TITLE
How to prevent click outside?

### ISSUE THREAD

**Issue by rusikf** [fundon/vue-click-outside#24]
Hi, with this plugin we can capture outside events, but cannot prevent it like event.preventDefault

Is it possible in vue.js way ?

**Comment by naquiroz** [comment]
I don't think it is the intended behavior, as it is not an event. But maybe I'm wrong.
